webapp/channels/src/components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -279,45 +339,59 @@ function createOffTopicIntroMessage(channel: Channel, centeredIntro: string, sta
 
     const channelInviteButton = (
         <AddMembersButton
-            setHeader={setHeaderButton}
             totalUsers={totalUsers}
             usersLimit={usersLimit}
             channel={channel}
             pluginButtons={<PluggableIntroButtons channel={channel}/>}
         />
     );
 
+    if (inviteUsers) {
+        actionButtons = (
+            <div className='channel-intro__actions'>
+                {actionButtons = channelInviteButton}
+            </div>
+        );
+    } else {
+        actionButtons = (
+            <div className='channel-intro__actions'>
+                {createFavoriteButton(isFavorite, toggleFavorite)}
+                {setHeaderButton}
+                {createNotificationPreferencesButton(channel, currentUser)}
+            </div>
+        );
+    }
+
     return (
         <div
             id='channelIntro'
             className={'channel-intro ' + centeredIntro}
         >
+            <ChannelIntroPublicSvg/>
             <h2 className='channel-intro__title'>
-                <FormattedMessage
-                    id='intro_messages.beginning'
-                    defaultMessage='Beginning of {name}'
-                    values={{
-                        name: channel.display_name,
-                    }}
-                />
+                {channel.display_name}
             </h2>
-            <p className='channel-intro__content'>
+            <div className='channel-intro__text'>
```

Comment:
There's also an E2E test that uses `channel-intro__content` as well

Reply:
Updated the test files